Content.Shared/Anomaly/Effects/Components/FleshAnomalyComponent.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Content.Shared.Maps;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype.List;
+
+namespace Content.Shared.Anomaly.Effects.Components;
+
+[RegisterComponent]
+public sealed class FleshAnomalyComponent : Component
```

Comment:
VV everything here (also `NetworkedComponent`?)

Reply:
this doesn't need to be networked because all of the code is in the server.